src/hypothesis/searchstrategy/regex.py

```diff
@@ -20,13 +20,19 @@
 import re
 import sys
 import operator
-import sre_parse as sre
+import sre_parse
 
 import hypothesis.strategies as st
 from hypothesis import reject
 from hypothesis.internal.compat import PY3, hrange, hunichr, text_type, \
     int_to_byte
 
+if False:
+    from typing import Any, Set, Text  # noqa
+
+# TODO: contribute stubs with sre_parse attributes to Typeshed
```

Comment:
TODO: contribute a complete stub for `sre_constants` to typeshed, and import constants from there instead of `sre_parse`